addons/mail/models/mail_thread.py

```diff
@@ -1887,8 +1887,8 @@ def _message_post_after_hook(self, message, msg_vals):
     # MESSAGE POST TOOLS
     # ------------------------------------------------------
 
-    def message_post_with_view(self, views_or_xmlid, **kwargs):
-        """ Helper method to send a mail / post a message using a view_id to
+    def _send_message_with_view(self, views_or_xmlid, message_log=False, **kwargs):
```

Comment:
Don't like the naming ... maybe def _message_compose_with_view ? as most cases finish on a composer, this is not necessarily a post or a log.